synapse/tests/test_cortex.py

```diff
@@ -1381,3 +1381,8 @@ def test_cortex_int_indx(self):
             self.len(1, core.eval('teststr +teststr:tick<201808021202'))
             self.len(0, core.eval('teststr +teststr:tick<201808021201'))
             self.len(0, core.eval('teststr +teststr:tick<201808021200'))
+
+            list(core.eval('[testint=99999]'))
+            self.len(1, core.eval('testint<=20'))
+            self.len(1, core.eval('testint>20'))
```

Comment:
Do we want to add a `>=` test here as well, just  ensure we're well behaved at that boundary?

Reply:
ok